### PR TITLE
Add support for getting the window from UIWindowSceneDelegate instead of UIApplicationDelegate

### PR DESCRIPTION
**Pull request by antoinelamy**
The current release of MCKillSwitch crash on start when integrated into a project that use a scene delegate. The UIWindow reference must be retrieved from the scene delegate instead of the traditional app delegate in those projects, this PR address this issue.

Note: The code only consider the first connected scene as supporting multiple windows would require a more in-depth refactoring.

### CODE REVIEW

File: MCKillSwitch/MCKillSwitchAlert.m [hunk 1]
```diff
@@ -150,14 +150,32 @@ - (void)hideAlertWithCompletion:(MCKillSwitchAlertBlock)completion {
...
         topMostViewController = topMostViewController.presentedViewController;
     }
     
     return topMostViewController;
 }
 
++ (UIViewController *)rootViewController {
+    UIViewController *rootViewController;
+    if (@available(iOS 13.0, *)) {
+        UIScene *scene = [[UIApplication.sharedApplication.connectedScenes allObjects] firstObject];
```
Comment (flambert) [inline MCKillSwitch/MCKillSwitchAlert.m:164]: Go all the way: `UIApplication.sharedApplication.connectedScenes.allObjects.firstObject`

Comment (antoinelamy) [reply MCKillSwitch/MCKillSwitchAlert.m:164]: Dots, dots everywhere! :allthethings:

Dire que ça existait même pas il y a pas si longtemps...

Comment (flambert) [reply MCKillSwitch/MCKillSwitchAlert.m:164]: Y a 8 ans...

Comment (antoinelamy) [reply MCKillSwitch/MCKillSwitchAlert.m]: Fixed :oldtimeyparrot:

**Pull request merged**